oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -499,6 +499,8 @@ private void configureBuilder(DocumentNodeStoreBuilder<?> builder) {
                 setDocStoreEmbeddedVerificationFeature(docStoreEmbeddedVerificationFeature).
                 setThrottlingEnabled(config.throttlingEnabled()).
                 setFullGCEnabled(config.fullGCEnabled()).
+                setFullGCIncludePaths(config.fullGCIncludes()).
```

Comment:
Please consider adding a check and WARN message if a path is part of both include and exclude lists. 
I am thinking exclude would take priority in this case and the path would be excluded, but it is something we can discuss if you think it is relevant.

Reply:
That's a good catch. I have tested with a path in both the `include` & `exclude` list and it turns out that in that case, we won't be fetching any document from db.
cc @stefan-egli

Reply:
I have tested on local and in case a path is present in both `include` & `exclude`, it won't be selected.

Reply:
Configuring the identical path in both include and exclude seems weird - but the code does behaves correctly, namely exclude has precedence, so it's "as if that path wasn't there". But I don't think we need to add an extra warn for this ..